in_text/text_00_parent.yml

```diff
@@ -42,3 +42,11 @@ entities:
     data-name: Parent metadata landing page
     data-description: >-
       This is the landing page for four other data release units, described in the attributes section
+
+cross-cites:
+  -
+    authors: ["Samantha K. Oliver", " Margaux Sleckman", "Alison P. Appling", "Hayley Corson-Dosch", "Theodore Thompson", "Lauren Koenig", "Ellie White", "David Watkins", "Lindsay Platt", "Julie Padilla", "Jeffrey M. Sadler"]
+    title: >-
+        Data to support water quality modeling efforts in the Delaware River Basin
+    pubdate: 2022
+    link: https://doi.org/10.5066/P9GUHX1U
```

Comment:
Should we be cross-referencing the base data release?

Reply:
```suggestion
cross-cites:
  -
    authors: ["Samantha K. Oliver", " Margaux Sleckman", "Alison P. Appling", "Hayley Corson-Dosch", "Jacob A. Zwart", "Theodore Thompson", "Lauren Koenig", "Ellie White", "David Watkins", "Lindsay Platt", "Julie Padilla", "Jeffrey M. Sadler"]
    title: >-
        Data to support water quality modeling efforts in the Delaware River Basin
    pubdate: 2022
    link: https://doi.org/10.5066/P9GUHX1U
```

Reply:
sure if it's not too hard to get middle initials. no big deal if we don't have them though